.github/workflows/build-wheels.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         env:
           CIBW_BUILD_VERBOSITY: 1
           CIBW_SKIP: cp27-* cp35-* cp36-* cp311-* pp* *musllinux*
-          CIBW_ARCHS_MACOS: "x86_64 arm64"
+          CIBW_ARCHS_MACOS: "universal2"
```

Comment:
Any reason for this change? I did it the other way to have smaller wheels to download.

Reply:
It uses the same name for x86 and arm wheels. Will need to fix and revert.